src/sentry/logging/handlers.py

```diff
@@ -0,0 +1,27 @@
+"""
+sentry.logging.handlers
+~~~~~~~~~~~~~~~~~~~~~~~
+:copyright: (c) 2010-2016 by the Sentry Team, see AUTHORS for more details.
+:license: BSD, see LICENSE for more details.
+"""
+
+import logging
+
+
+class StructLogHandler(logging.StreamHandler):
+    def __init__(self, *args, **kwargs):
+        from structlog import get_logger
+        from sentry.runner.initializer import configure_structlog
+        super(StructLogHandler, self).__init__(*args, **kwargs)
+        configure_structlog()
```

Comment:
Why is `configure_structlog()` being called twice? Here, and inside `initialize_app`.